drivers/mbox/CMakeLists.txt

```diff
@@ -14,3 +14,5 @@ zephyr_library_sources_ifdef(CONFIG_MBOX_NRF_VEVIF_LOCAL mbox_nrf_vevif_local.c)
 zephyr_library_sources_ifdef(CONFIG_MBOX_NRF_VEVIF_REMOTE mbox_nrf_vevif_remote.c)
 zephyr_library_sources_ifdef(CONFIG_MBOX_NRF_BELLBOARD_LOCAL mbox_nrf_bellboard_local.c)
 zephyr_library_sources_ifdef(CONFIG_MBOX_NRF_BELLBOARD_REMOTE mbox_nrf_bellboard_remote.c)
+zephyr_library_sources_ifdef(CONFIG_MBOX_NRF_VEVIF_EVENT_LOCAL mbox_nrf_vevif_event_local.c)
+zephyr_library_sources_ifdef(CONFIG_MBOX_NRF_VEVIF_EVENT_REMOTE mbox_nrf_vevif_event_remote.c)
```

Comment:
I'm thinking if it would be clearer to rename all local/remote labels for VEVIF and BELLBOARD to something more descriptive like "sender/receiver", "source/destination", "trigger/recipient"